<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QLineEdit" name="input_folder_path">
         <property name="toolTip">
          <string>The Destination For The Output Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_Button">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="input_file_name">
       <property name="toolTip">
        <string>The Rended Image File Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="framePaddingLabel">
       <property name="text">
        <string>Frame Padding</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="input_frame_padding">
         <property name="toolTip">
          <string>The Number Of 0 Added To The File Name Before The Current Frame Number</string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Digits</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="channelsLabel">
       <property name="text">
        <string>Channels</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="input_channels">
       <property name="toolTip">
        <string>The Channle That Will Be Used For The Rended Image</string>
       </property>
       <item>
        <property name="text">
         <string>rgb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rgba</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="fileTypeLabel">
       <property name="text">
        <string>File Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="fileTypeComboBox">
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpeg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tiff</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="enableViewsLabel">
       <property name="text">
        <string>Enable Views</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="enableViewsCheckBox"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="build_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Yep You Guessed It This Builds The Ouput</string>
       </property>
       <property name="text">
        <string>Build</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="active_view_checkBox">
       <property name="toolTip">
        <string>Will reset the active view to the write node if active</string>
       </property>
       <property name="text">
        <string>Active View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Layer_Order_UI" name="Layers_Order_Widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Layer_Order_UI</class>
   <extends>QWidget</extends>
   <header>layer_order_ui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
